types.go

```diff
@@ -0,0 +1,80 @@
+package cedar
+
+import (
+	"time"
+
+	"github.com/cedar-policy/cedar-go/types"
+)
+
+// Cedar data types
+type Boolean = types.Boolean
+type Datetime = types.Datetime
+type Decimal = types.Decimal
+type Duration = types.Duration
+type EntityUID = types.EntityUID
+type IPAddr = types.IPAddr
+type Long = types.Long
+type Record = types.Record
+type RecordMap = types.RecordMap
+type Set = types.Set
+type String = types.String
+
+// Other Cedar types
+type Entities = types.Entities
+type Entity = types.Entity
+type EntityType = types.EntityType
+type Pattern = types.Pattern
+type Wildcard = types.Wildcard
+
+// cedar-go types
+type Value = types.Value
+
+// Constants
+const (
+	True  = types.True
+	False = types.False
+)
+
+const (
+	DecimalPrecision = types.DecimalPrecision
+)
+
+// Constructors
```

Comment:
Check how this reads in pkgsite.  It might be nice to either copy the docs, or provide a [link] to the docs for each of the functions in the types package.

Reply:
Good call. I've fleshed out the docs for the rest of the functions.